addons/mass_mailing/tests/test_mailing_internals.py

```diff
@@ -370,3 +370,26 @@ def test_mailing_shortener(self):
                     link_info,
                     link_params=link_params,
                 )
+
+class TestMailingScheduleDateWizard(MassMailCommon):
```

Comment:
Would move it to test_mailing_mailing_schedule_date.py file (one file per main model to test, so that we know we have test for that wizard)

Reply:
Moved to the new test_mailing_mailing_schedule_date.py file.